ocaml/xapi/xapi_pusb.ml

```diff
@@ -74,20 +80,21 @@ let start_thread f =
           ()
       done) ())
 
-let scan ~__context =
+let scan ~__context ~host =
   (* notify that scan is required. *)
   Mutex.execute mutex (fun () ->
+      host_ref := host;
```

Comment:
again, no need for this. You're protecting against the call be routed to the wrong host, and if _that_ logic went wrong, xapi would fall apart completely :-)

Reply:
updated.